config/ucx.conf

```diff
@@ -2,6 +2,7 @@
 CPU model=Grace
 UCX_REG_NONBLOCK_MEM_TYPES=host
 UCX_IB_MLX5_DEVX_OBJECTS=
+UCX_GDR_COPY_BW=get_dedicated:15GBs,put_dedicated:15GBs
```

Comment:
is there a scenario where shared bw is not 0 ?

Reply:
That's the default scenario (see here https://github.com/openucx/ucx/pull/10112/files#diff-d74cb886a55c337f1b0cebb7a424dd4da3e7a88745b3e6282aaf3a1a2bd54a04R5)

Reply:
According to `UCX_GDR_COPY_BW=0MBs,get_dedicated:15GBs,put_dedicated:15GBs`, seems shared is 0?

Reply:
Yes, that's right, shared is 0 and dedicated for both put and get is 15GBs

Reply:
so if shared is always 0 for gdrcopy, maybe we need to have configuration only for dedicated bw?

Reply:
Shared is 0 for GH systems but it is not zero for common case: https://github.com/openucx/ucx/pull/10112/files#diff-f52d5bd2e1444dd959c6c78c1e2c7db60af22046db17517e03e62e65b9b8ef9fR29